integration_tests/__init__.py

```diff
@@ -104,3 +106,10 @@ def logout(self):
         expected = ('Clearing credentials for Ubuntu One SSO.\n'
                     'Credentials cleared.\n')
         self.assertEqual(expected, output)
+
+    def register_name(self, name):
+        process = pexpect.spawn(self.snapcraft_command,
+                                ['register-name', name])
```

Comment:
This has been defined by Martin and Mark, I wouldn't change it without feedback.
I think the rationale is that /names/ matters in the snappy world, they are defined by series and can change owner from one series to the others, there can be disputes, revocations and so on.
So, register-name carry that, a bare register will be less clear. But that's just my understanding.